pykanidm/kanidm/__init__.py

```diff
@@ -165,6 +165,7 @@ async def _call(
                 elif "authorization" not in headers:
                     logging.debug("Setting auth headers as Authorization not in keys")
                     headers.update(self._token_headers)
+            logging.debug("headers --> %s" % headers)
```

Comment:
I specifically wasn't logging this because it leaks auth tokens, and that's bad...

Reply:
Fixed!